PythonExample/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Can we just ignore the .idea folder completely?  I don't think this is needed in the repository.